modules/client-configuration/pages/clients-rocky.adoc

```diff
@@ -0,0 +1,137 @@
+[[clients-rocky]]
+= Registering {rocky} Clients
+
+This section contains information about registering Salt clients running {rocky} operating systems.
+
+[NOTE]
+====
+Traditional clients are not available on {rocky}{nbsp}8.
+{rocky}{nbsp}8 clients are only supported as Salt clients.
+====
+
+[NOTE]
+====
+Registering {rocky} clients to {productname} is tested with the default SELinux configuration of ``enforcing`` with a ``targeted`` policy.
+You do not need to disable SELinux to register {rocky} clients to {productname}.
+====
+
+
+
+== Add Software Channels
+
+Before you can register {rocky} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
+
+ifeval::[{suma-content} == true]
+
+The products you need for this procedure are:
+
+[[rocky-channels-wizard]]
+[cols="1,1", options="header"]
+.CentOS Products - WebUI
+|===
+| OS Version | Product Name
+| {rocky} 8 | {rocky} 8 x86_64
+|===
+
+include::snippets/addchannels_vendor_webui.adoc[]
+
+Alternatively, you can add channels at the command prompt.
+The channels you need for this procedure are:
+
+[[rocky-channels-suma-cli]]
+[cols="1,1", options="header"]
+.CentOS Channels - CLI
+|===
+| OS Version | Base Channel
+| {rocky} 8 | rocky8-x86_64
+|===
+
+include::snippets/addchannels_vendor_cli.adoc[]
+
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+The channels you need for this procedure are:
+
+[[rocky-channels-uyuni-cli]]
+[cols="1,1,1,1", options="header"]
+.CentOS Channels - CLI
+|===
+| OS Version | Base Channel | Client Channel       | AppStream Channel
+| {rocky} 8 | rocky8      | rocky8-uyuni-client | rocky8-appstream
```

Comment:
I just noticed a typo here. I need to fix it. It's `rockylinux` and not `rocky`. Preparing a PR ASAP.